codice/ddf
# DDF-519 DDF-1460 DDF-2090 GeoJSON multivalued attribute support

#### What does this PR do?
This PR adds support for multivalued attributes with the GeoJSON transformers.  It also replaces the MetacardTypeRegistry with MetacardType OSGi services and fixes a bug with geometry collection handling.

#### Who is reviewing it
@rzwiefel 
@djblue 

#### Choose 2 committers to review/merge the PR
@kcwire
@pklinef

#### How should this be tested?
Using any metacard with multivalued attributes, export and import the metacard using the GeoJSON transformer and verify the multivalued attributes are intact.

#### Any background context you want to provide?

#### What are the relevant tickets?
https://codice.atlassian.net/browse/DDF-519
https://codice.atlassian.net/browse/DDF-1460
https://codice.atlassian.net/browse/DDF-2090

#### Screenshots (if appropriate)

#### Checklist:
- [n/a] Documentation Updated
- [n/a] Update / Add Integration Tests
